src/img/ores/bronze-ore.piskel

```diff
@@ -1 +1 @@
-{"modelVersion":2,"piskel":{"name":"bronze-ore","description":"","fps":12,"height":24,"width":24,"layers":["{\"name\":\"Layer 1\",\"opacity\":1,\"frameCount\":1,\"chunks\":[{\"layout\":[[0]],\"base64PNG\":\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAABFUlEQVRIS7WVSw7CMBBD6W0QYsdNOCg3YYcQtwFNkSvH9eTTql2B0vrF45lkOh38TJn+5Xr+xtr79Unf6dmb/RjiLLAVlALut//S4zkbKZ4R2AoQuw9xCOO3AnshFhDbVUH8h6tdABVj4C4AwmUABHc7yMQ5aM5mqEQsHuHqbl3ovTMyh4zOQS8yhLtJ17FWc1MFOHGdDXbnXK0AbrBc6K6zAGNHBaBWDp1mzUqdAbIAINBqUc5BXehaQKoOuFxZmVqlWo4KPoO0LbNZyNqXwy4A2E1v92hJ+PsiA7zI94CWRF1w6Ahc52fJQDvEHRs1V+7usA4Y5G41F6huTqd66L5VZ+GqdegNATSrlni8vwmgZan9/wHkxAkoX3dEowAAAABJRU5ErkJggg==\"}]}"]}}
\ No newline at end of file
+{"modelVersion":2,"piskel":{"name":"stone-bronze-ore","description":"","fps":12,"height":24,"width":24,"layers":["{\"name\":\"Layer 1\",\"opacity\":1,\"frameCount\":1,\"chunks\":[{\"layout\":[[0]],\"base64PNG\":\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAABdklEQVRIS+2UvUoDQRSFJ4WkUIxvoIKFD5LGysJXsNhUCgb0ERQixG4LXyGFlU0ewkZYUDCCrYighVisnBvPemZn9qcJKDhNNjOb79x7zp103IJXZ8F89y/Q6PCvsSivKbW2yKYODHw2One9lWXTeH17d3tLU3uefPbd8fBItQNenUAOsML4DOLlw3rQ1Ok4xZ7HrBIw+MvTXQHZ33z0gNPurn3vf1wV3aCArYNrTyQm4FVeBuHX2AOYn1QWywpuWSC/v9gxb7lQlYrwWbPguzhLkqSyA4PrUiGGzHMIYI8WaSaahXZgArfPc0S2Ou+CkFioOEc27IqZaA6eQJraFLhZdhNMCEOOZcJcaFulwMmh+WdVqa+0oWyf2haDBzPrnMvRBYGxcFVslG2Y5lr356KX70IwRbSJY6hVK1wzQccQG27P2t0DQJPBwKtMbWPlFAcYq3zJYhZpwTlEdAEUg3+Do7xWf3bBSIUblZwmgRbs+lf+vsAXSn2tGWpeb/IAAAAASUVORK5CYII=\"}]}"]}}
```

Comment:
Since the file names have been updated (thank you!), please make sure to update the `name`s in the Piskel source files to match.

Reply:
oh yeah good catch